tests/framework/CodeInterfaceTests/raven_running_raven_internal_models/test_rom_trainer.xml

```diff
@@ -148,7 +148,7 @@
     </PointSet>
     <PointSet inputTs="2" name="Pointset_from_database_for_rom_trainer">
       <Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
-      <Output>CladTempThreshold</Output>
+      <Output>CladTempThreshold,time</Output>
```

Comment:
why does the variable time need to be added?

Reply:
removed...but the test does not work still...